README.md

```diff
@@ -125,15 +127,9 @@ following steps:
 git clone https://github.com/docker/docker-bench-security.git
 cd docker-bench-security
 docker build --no-cache -t docker-bench-security .
-docker run -it --net host --pid host --cap-add audit_control \
-    -e DOCKER_CONTENT_TRUST=$DOCKER_CONTENT_TRUST \
-    -v /var/lib:/var/lib:ro \
-    -v /var/run/docker.sock:/var/run/docker.sock:ro \
-    -v /usr/lib/systemd:/usr/lib/systemd:ro \
-    -v /etc:/etc:ro --label docker_bench_security \
-    docker-bench-security
 ```
 
+followed but an appropiate `docker run` command as stated above
```

Comment:
spelling: I think this should say "followed by an appropriate"

Reply:
so true